### ISSUE TITLE
[Bug][a11y][VoiceOver]: Range Slider handles have no accessibility name  

### ISSUE THREAD

**Issue by majornista** [adobe/spectrum-web-components#3326]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Slider, SliderHandle

### Expected behavior

Using Safari with VoiceOver on either macOS or iOS, the slider handles for Range Sliders examples on https://opensource.adobe.com/spectrum-web-components/components/slider-handle/ should announce with the `label` provided for `sp-slider` and the `label` provided for each `sp-slider-handle`. The example below should have two inputs announced as "5, Output Levels Minimum, slider" and "250, Output Levels Maximum, slider."

```html
<sp-slider variant="range" step="1" min="0" max="255">
    Output Levels
    <sp-slider-handle
        slot="handle"
        name="min"
        label="Minimum"
        value="5"
    ></sp-slider-handle>
    <sp-slider-handle
        slot="handle"
        name="max"
        label="Maximum"
        value="250"
    ></sp-slider-handle>
</sp-slider>
```

The labelling seems to be working as expected with Chrome, Edge and Firefox.

### Actual behavior

Using Safari with VoiceOver on either macOS or iOS, the slider handles for Range Sliders examples on https://opensource.adobe.com/spectrum-web-components/components/slider-handle/ announce without any accessibility name. The example below has two inputs announced as "5, slider" and "250, slider," without context.

```html
<sp-slider variant="range" step="1" min="0" max="255">
    Output Levels
    <sp-slider-handle
        slot="handle"
        name="min"
        label="Minimum"
        value="5"
    ></sp-slider-handle>
    <sp-slider-handle
        slot="handle"
        name="max"
        label="Maximum"
        value="250"
    ></sp-slider-handle>
</sp-slider>
```

Also, note that the multi-handle storybook examples do not provide a `label` for each of the `sp-slider-handle` elements: https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/slider--two-handles

### What browsers are you seeing the problem in?

Safari

### How can we reproduce this issue?

3. Scroll to '....'
4. Check console
5. See error

**Comment by najikahalsema** [comment]
SWC-270

**Comment by caseyisonit** [comment]
Hello @jnurthen in order to help us triage your issue, [can you please review our bug severity classification guide](https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md#issue-severity-classification) and provide us with the appropriate severity classification? Please update the label to the correct severity level and remove `SEV missing` label.

If we don't hear from you in 2 weeks, we will close this issue as not planned.

<img width="1024" height="1028" alt="Image" src="https://github.com/user-attachments/assets/d5fd78ee-6e8e-4967-97f3-4e4198b79f13" />
